docs/qa/v034/img/v034_report_tabbed.txt

```diff
@@ -0,0 +1,64 @@
+Experiment ID: 423ebe72-d27c-45b5-bedc-9874547fe9c1 │Experiment ID: 3b1f0dd9-b66a-4edc-8f64-2ba134a1d022 │Experiment ID: a8ad4147-9453-4147-81be-367a8d386b4a
+                                                    │                                                    │
+        Connections: 4                              │        Connections: 8                              │        Connections: 16
+        Rate: 20                                    │        Rate: 20                                    │        Rate: 20
+        Size: 1024                                  │        Size: 1024                                  │        Size: 1024
+                                                    │                                                    │
+        Total Valid Tx: 144                         │        Total Valid Tx: 309                         │        Total Valid Tx: 632
```

Comment:
Am I reading this correctly that, in 90 seconds of load with 4/8/16 connections, sending transactions at 20 per second, we only recorded 144/309/632 valid transactions?

With these numbers, it only looks as though 2 seconds' worth of transactions were sent in each of the above experiments? i.e.
- 144 / (4 x 20) = 1.8s
- 309 / (8 x 20) = 1.93s
- 632 / (16 x 20) = 1.98s

If the transactions were, in fact, sent across 90 seconds, it would mean that the rates would be:
- 144 / 90 = 1.6 tx/sec
- 309 / 90 = 3.43 tx/sec
- 632 / 90 = 7 tx/sec

Am I missing something here?

Reply:
That is spot on. William noticed the same.
We had already realized that there seems to be a "factor of 50" we are missing somewhere.
I've been digging for the last couple of hours, and I haven't found the reason for this divergence. I've re-checked the prometheus data both on `v0.34` and `v0.37`, and it corroborates the report shown here. So somehow the load produced was less than configured. I'm now looking at the code of `tm-load-test` to see if we missed something there, but can't find anything substantial, only that the `send-period` affects the rate (the rate doesn't seem to be "per-second", but "per `send-period`") but the default is 1, and I don't think we set it to 50... even in that case, prometheus wouldn't match, as it shows a constant flow of transactions every second, not every 50s.
I'll discuss this with @williambanfield tomorrow when he's back.

Reply:
This ended up being #9539, so I guess I can resolve this